src/main/java/com/puppycrawl/tools/checkstyle/utils/ScopeUtil.java

```diff
@@ -70,18 +70,17 @@ public static Scope getSurroundingScope(DetailAST node) {
              token != null;
              token = token.getParent()) {
             final int type = token.getType();
-            if (type == TokenTypes.CLASS_DEF
-                || type == TokenTypes.INTERFACE_DEF
-                || type == TokenTypes.ANNOTATION_DEF
-                || type == TokenTypes.ENUM_DEF) {
+            if (TokenUtil.isOfType(type, TokenTypes.CLASS_DEF,
```

Comment:
I'd like to have a dedicated method for Class/Interface/Enum/Annotation checking.
Something like `TokenUtil.isTypeDeclaration(tokenType)`

Reply:
can we do it in a separate PR?

Reply:
Sure.

Reply:
please open a new issue for it. i'll work on it as soon as it get approved.